### PR TITLE
Format list of assets into a table

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
Should we merge the 4 tables into one with each type (Miniforge, Miniforge-pypy3, Mambaforge, Mambaforge-pypy3) getting a column each?

**Comment by SylvainCorlay** [comment]
> Should we merge the 4 tables into one with each type (Miniforge, Miniforge-pypy3, Mambaforge, Mambaforge-pypy3) getting a column each?

Why not, although this may be a bit dense for newcomers, and we would need to make a placeholder text for the download link to save some space.

**Comment by SylvainCorlay** [comment]
I think the current version is clearer and a more dense single table.

File: README.md [hunk 1]
```diff
@@ -14,41 +14,50 @@ Miniforge installers are available here: https://github.com/conda-forge/miniforg
...
-- [`linux-x86_64`](https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh) (also called `amd64`)
-- [`osx-x86_64`](https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-x86_64.sh)
-- [`osx-arm64`](https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-arm64.sh) (Apple Silicon)
-- [`win-x86_64`](https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Windows-x86_64.exe)
+
+| OS      | Architecture          | Download  |
+| --------|-----------------------|-----------|
+| Linux   | x86_64 (amd64)        | https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh |
+| Linux   | aarch64 (arm64)       | https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh |
+| Linux   | ppc64 (POWER8/9)      | https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-ppc64le.sh |
```
Comment (isuruf) [inline README.md:46]:
```suggestion
| Linux   | ppc64le (POWER8/9)      | https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-ppc64le.sh |
```

**Comment by isuruf** [comment]
> we would need to make a placeholder text for the download link to save some space.

we should do that anyway as some rows are wrapped into two lines at the moment

**Review by isuruf — APPROVED** [review]

**Pull request merged**